modules/services/repositories/src/main/java/au/com/shiftyjelly/pocketcasts/repositories/support/Support.kt

```diff
@@ -270,6 +272,13 @@ class Support @Inject constructor(
                 }
             }
 
+            runCatching {
+                val features = Feature.entries.map { "${it.key}: ${FeatureFlag.isEnabled(it)}" }
+                output.append("Feature flags").append(eol)
+                features.forEach { output.append(it).append(eol) }
+                output.append(eol)
```

Comment:
The last eol isn't needed as there is one before the next section, so it adds two lines.

Reply:
Ah, haven't noticed. I'll make a follow-up PR.